templates/deployment-flower.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: flower
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: flower
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: flower
+        image: eanylin/airflow-helm:1.8.1-1.6.2
```

Comment:
This needs to point to a publicly available image.  This would require the user build it locally.

For the interim, we should commit the necessary `Dockerfile` to github.com/att-comdev/dockerfiles/airflow